test_mapharma.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import scraper
+
+from datetime import datetime
+from multiprocessing import Pool
+
+from scraper.mapharma import centre_iterator as mapharma_centre_iterator
+from scraper.mapharma import fetch_slots as mapharma_fetch_slots
+from scraper.pattern.scraper_result import ScraperResult
+from scraper.pattern.scraper_request import ScraperRequest
+from utils.vmd_logger import enable_logger_for_production, enable_logger_for_debug
+
+logger = logging.getLogger('scraper')
+today = datetime.now().strftime('%Y-%m-%d')
+
+def mapharma_scrape(centre):
+    vaccine_center = centre["nom"]
+    com_insee = centre["com_insee"]
+    rdv_site_web = centre["rdv_site_web"]
+    logger.info("Vaccine Center {0}: looking for an appointment".format(vaccine_center))
```

Comment:
f-strings everywhere \o/
```suggestion
    logger.info(f"Vaccine Center {vaccine_center}: looking for an appointment")
```